build.xml

```diff
@@ -13,6 +13,20 @@
     <target name="standards" depends="phplint,phpcs-fixer,phpcs,phpmd,phpstan,twig-lint,yaml-lint,eslint-check,standards-packages" description="Checks coding standards in both project-base and packages."/>
     <target name="standards-diff" depends="phplint-diff,phpcs-fixer-diff,phpcs-diff,phpmd-diff,phpstan,twig-lint-diff,yaml-lint,eslint-check-diff,standards-packages" description="Checks coding standards on changed files in project-base and all files in packages."/>
 
+    <target name="clean" description="Cleans up directories with cache and scripts which are generated on demand.">
+        <delete failonerror="false" includeemptydirs="true">
+            <fileset dir="${path.var}/cache/">
+                <exclude name="/" />
+            </fileset>
+            <fileset dir="${path.web.scripts}/">
+                <exclude name="/" />
+            </fileset>
+        </delete>
+        <delete failonerror="false" includeemptydirs="true" quiet="true">
+            <filelist dir="${path.root}" files="migrations_config.yml"/>
```

Comment:
You don't have to copy-paste the target in order to append one more call, see https://stackoverflow.com/questions/12715855/in-a-phing-build-file-when-i-import-another-build-file-and-then-override-a-targ

Also, why do we need to remove the file?

Reply:
Do we really need to *always* install migrations in the default order? I think it would be better to test the real behavior of the app. We can always manually remove the file.